exercises/practice/yacht/src/yacht.cob

```diff
@@ -5,6 +5,5 @@
        01 WS-RESULT PIC 99 VALUE 0.
        01 WS-CATEGORY PIC X(15).
        01 WS-DICE PIC 9(5).
-	   PROCEDURE DIVISION.
-	   YACHT.
-	   
\ No newline at end of file
+          PROCEDURE DIVISION.
+          YACHT.
```

Comment:
It might be important to leave a new line at the end of the file. gnuCOBOL will complain if it's not there.